modules/javafx.graphics/src/main/java/javafx/css/CssMetaData.java

```diff
@@ -327,5 +328,35 @@ public int hashCode() {
             .append("}").toString();
     }
 
-
+    /**
+     * Utility method which combines {@code CssMetaData} items in one immutable list.
+     * <p>
+     * The intended usage is to combine the parent and the child CSS meta data for
+     * the purposes of {@code getClassCssMetaData()} method, see for example {@link Node#getClassCssMetaData()}.
+     * <p>
+     * Example:
+     * <pre>
+     * private static final List&lt;CssMetaData&lt;? extends Styleable, ?>> STYLEABLES = CssMetaData.combine(
+     *      &lt;Parent>.getClassCssMetaData(),
+     *      STYLEABLE1,
+     *      STYLEABLE2
+     *  );
+     * </pre>
+     * This method returns an instance of {@link java.util.RandomAccess} interface.
+     *
+     * @param list the css metadata items, usually from the parent, must not be null
```

Comment:
Two minor things:
1. I think you should either spell out the actual class name `CssMetaData`, or capitalize the `CSS`.
2. Maybe change the parameter name to `inherited`, `inheritedMetaData`, or something like this. I'd also remove the `usually from the parent` part, as this leaves the option that it might come from somewhere else (which isn't intended).